coalib/coala_main.py

```diff
@@ -31,7 +31,7 @@ def do_nothing(*args):
 
 
 STR_ENTER_LETTER = 'Enter letter (Ctrl-{} to exit): '.format(
-    'Z' if platform.system() == 'Windows' else 'D')
+    'Z-Enter' if platform.system() == 'Windows' else 'D')
```

Comment:
`Ctrl-Z-Enter` would look like we need to press all the 3 keys simultaneously. But iirc, The proper way to send an EOF signal in Windows is to press `Ctrl-Z` followed by `Enter`. So something like, `Ctrl-Z followed by Enter` would seem more appropriate.